packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -654,7 +654,8 @@ export default class RestApi {
     api.get(
       '/dashboard',
       wrap(async (req, res) => {
-        res.json(await getDashboardStats(app))
+        // returns an array because the dashboard data will be consumed by the UI as a collection
```

Comment:
I do not want this change, the REST API currently returns array for collections but plain object for simple records.

This may be change in the future for the whole API if we think that's the best approach but I do not want to break this homogeneity simply for an API client.

Reply:
@P4l0m4  @OlivierFL  @ByScripts

Reply:
I also don't think the API should change the way it works just to fit the client needs.

We can adapt the code to handle the specific case of single-record requests.